src/pypi_attestations/_cli.py

```diff
@@ -267,7 +272,12 @@ def _verify(args: argparse.Namespace) -> None:
             _die(f"Invalid attestation ({file_path}): {validation_error}")
 
         try:
-            attestation.verify(verifier, pol, file_path)
+            dist = Distribution.from_file(file_path)
+        except ValidationError as e:
+            _die(f"Invalid Python package distribution: {e}")
```

Comment:
`from_file` might throw an `OSError`. Do we want to not handle it here, and just redirect it to the user?

Reply:
Yeah, I wasn't sure what to do here -- I don't love that we're exposing `ValidationError` either but that can't really be avoided while using `pydantic` 🙂 

Given that this is a helper and our main consumers (PyPI and the public action) should be able to construct directly, I'm inclined to say that we can leave it as-is. But I'll add a test for the `OSError` case as well.

Reply:
> edit2: Also we could have a ValueError from ultranormalize

This one should be wrapped, since the Pydantic `@field_validator` will catch `ValueError`s and turn them into `ValidationError`s.